keps/sig-api-machinery/3488-cel-admission-control/README.md

```diff
@@ -2401,7 +2401,13 @@ in back-to-back releases.
 - support the list of features mentioned under phrase 2
 
 ## GA
-- complete type check for CRD and aggregated types
+- Complete type check for CRD and aggregated types
+- Scalability evaluation. Evaluate ValidatingAdmissionPolicy scalability including how many validators it could run, runtime cost evaluation, evaluating the reanable ResourceQuota for
+  ValidatingAdmissionPolicy, how much faster comparing with Webhook, the scale target, etc.
+- Fix the known issues along the way([variables should keep the type info](https://github.com/kubernetes/kubernetes/issues/120239), [variables should mark omitempty](https://github.com/kubernetes/kubernetes/issues/120491), [properly escape](https://github.com/kubernetes/kubernetes/issues/118230))
+- Define the mechanism for CEL library update policy after GA(version skew policy, the policy to update CEL library, etc)
```

Comment:
Let's also add a item to compare the CEL libraries with what is going to be provided for Declarative Validation. I'd like to make sure we don't GA this feature and then add a bunch of CEL library changes after it goes to GA..

Reply:
Added